src/Files.App (Package)/Package.appxmanifest

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think it's a problem, but the byte order mark has been gone.

Reply:
It woun't be problem because VS adds back.